Source/DataLayer/EfClasses/LibraryBook.cs

```diff
@@ -13,16 +13,18 @@ public class LibraryBook
 
         public bool IsDeleted { get; set; }
         public bool AbsentFromLastScan { get; set; }
-
+        
+        public DateTime? IncludedUntil { get; private set; }
         private LibraryBook() { }
-        public LibraryBook(Book book, DateTime dateAdded, string account)
+        public LibraryBook(Book book, DateTime dateAdded, string account, DateTime? includedUntil)
```

Comment:
includedUntil should be removed from the constructor. Use a setter like `SetAccount` below.